spec/inputs/http_poller_spec.rb

```diff
@@ -25,6 +25,10 @@
 
   subject { klass.new(default_opts) }
 
+  before do
+    #subject.register
```

Comment:
Oops, yeah, that wasn't a great idea. That's too high level a scope to call that since in some spots `subject` is an `Event` in others its an instance of the poller.